src/translations/nbTranslations.json

```
@@ -1023,7 +1023,7 @@
       "start_with_empty_form": "Start uten noen ferdig utfylte opplysninger.",
       "start_with_nfr_project": "Hvis du har finansiering fra Norges Forskningsråd (NFR), så kan du starte med den og få noen felt ferdig utfylt."
     },
-    "grant_id": "GrantID",
+    "grant_id": "TilskuddsID",
```

Samkjør med @torbjokv hva dette skal være: https://github.com/BIBSYSDEV/NVA-Frontend/pull/5166/files#diff-25b167578ee2e00fcc383fee9756f8e719d88efd889b99602abbb618f8ebcb7c
Etter begge PRs burde de bruke samme key tenker jeg
Har også et tredje forslag til teksten :D
```suggestion
    "grant_id": "Tilskudds-ID",
```

---

Det kommer rett fra Jan Erik at det skal være TilskuddsID :)

---

det som er faren med en dyslektisk PO 😂